File: ToggleApp/app/src/main/java/com/collinskandie/toggleapp/MainActivity.java
package com.collinskandie.toggleapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;
import android.widget.ToggleButton;

public class MainActivity extends AppCompatActivity {
    private ToggleButton toggleButton3,toggleButton4;
    private Button button;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        //on click listener.
        addListenerOnButtonClick();
    }

    private void addListenerOnButtonClick() {
        //getting the buttons from the layout.
        toggleButton3=(ToggleButton)findViewById(R.id.toggleButton3);
        toggleButton4=(ToggleButton)findViewById(R.id.toggleButton4);
        button=(Button)findViewById(R.id.button);

        button.setOnClickListener(new View.OnClickListener());
        @Override
                public void onClick(View View){
            StringBuilder result= new StringBuilder();
            result.append("ToggleButton1:").append(toggleButton3.getText());
            result.append("\n ToggleButton4").append(toggleButton4.getText());
            //display the message in toast
            Toast.makeText(getApplicationContext(), result.toString(),Toast.LENGTH_LONG).show();

        }
    }

}

File: messageapp/app/src/main/java/com/collinskandie/messageapp/MainActivity.kt
package com.collinskandie.messageapp

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnShowToast.setOnClickListener {

            Log.i("MainActivity","Button Was clicked")

            Toast.makeText(this, "Button was clicked", Toast.LENGTH_SHORT).show()
        }
        btnSendMessageToNextActivity.setOnClickListener {

           val message: String =userMessage.text.toString()
            Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
            Toast.makeText(this, "Sending...", Toast.LENGTH_SHORT).show()

            val intent = Intent( this, SecondActivity::class.java)
            intent.putExtra("user_message",message)
            startActivity(intent)



        }
    }
}